pkg/filterengine/filterengine.go

```diff
@@ -27,15 +27,11 @@ import (
 	"github.com/infracloudio/botkube/pkg/log"
 )
 
-var (
-	// DefaultFilterEngine contains default implementation for FilterEngine
-	DefaultFilterEngine FilterEngine
-)
-
 // FilterEngine has methods to register and run filters
 type FilterEngine interface {
 	Run(interface{}, events.Event) events.Event
 	Register(Filter)
+	RegisterMany([]Filter)
```

Comment:
Maybe `RegisterAll`? Don't have a strong opinion though

Reply:
🤔 I didn't like `RegisterMany` either, so I changed the `Register` interface method to `Register(...Filter)` and now it looks good to me, hopefully you'll also like it 🙂